api/app/tests/test_filtering.py

```diff
@@ -0,0 +1,154 @@
+import unittest
+
+from django.contrib.gis.geos import Point
+from django.utils.http import urlencode
+from rest_framework.test import APITestCase
+
+from tests.factories import SignalFactory
+
+IN_AMSTERDAM = (4.898466, 52.361585)
+N_RECORDS = 10
+
+SIGNAL_ENDPOINT = '/signals/auth/signal/'
+STATUS_ENDPOINT = '/signals/auth/status/'
+LOCATION_ENDPOINT = '/signals/auth/location/'
+
+
+class FilterTestDataMixin(APITestCase):
+    def setUp(self):
+        super()
+        signals = [SignalFactory.create() for i in range(N_RECORDS)]
+
+        # Change the coordinates, to be increasing in both longitude and latitude
+        self.dlon = 1e-3  # single increment in longitude
+        self.dlat = 1e-3  # single increment in latitude
+
+        for i, signal in enumerate(signals):
+            new_point = Point(IN_AMSTERDAM[0] + i * self.dlon,
+                              IN_AMSTERDAM[1] + i * self.dlat)
+            signal.location.geometrie = new_point
+            signal.location.save()
+
+    def _get_response(self, endpoint, payload):
+        path = endpoint + '?' + urlencode(payload)
+        return self.client.get(path)
+
+
+# -- refactor away repetitive code --
+
+
+class TestBboxFilter(FilterTestDataMixin, APITestCase):
```

Comment:
No need to double inherit from `APITestCase`. Maybe the best way is to inherit from `object` for the mixin class?

Reply:
You're right! No need for `object`.